### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by max-zviahintsev** [Creatiwity/gatsby-plugin-favicon#53]
The plugin worked normally but today it causes this error:
```
yarn add gatsby-plugin-favicon
yarn add v1.12.3
[1/4] Resolving packages...
error Command failed.
Exit code: 128
Command: git
Arguments: ls-remote --tags --heads git://github.com/Creatiwity/favicons-webpack-plugin.git
Directory: C:\server\z-client
Output:
fatal: unable to look up github.com (port 9418) (Either the application has not called WSAStartup, or WSAStartup failed. )
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Comment by julien1619** [comment]
Hi! Thanks for the report. Can you share your setup? Gatsby version, etc.

**Comment by max-zviahintsev** [comment]
Hi, Julien!
Nothing special.  All packages have the latest versions. I developed and deployed my app without a problem a week ago, but today I tried to update packages and make some changes but received mentioned error while updating. Then I deleted node_modules and yarn lock and run "yarn" command, but still, plugin threw the same error. I had to remove it to proceed.
Here is my package.json
```
  "dependencies": {
    "antd": "^3.9.3",
    "babel-plugin-import": "^1.9.1",
    "babel-plugin-styled-components": "^1.7.1",
    "delay": "^4.0.1",
    "gatsby": "^2.0.0",
    "gatsby-plugin-antd": "^2.0.2",
    "gatsby-plugin-google-analytics": "^2.0.7",
    "gatsby-plugin-less": "^2.0.5",
    "gatsby-plugin-react-helmet": "^3.0.2",
    "gatsby-plugin-sitemap": "^2.0.2",
    "gatsby-plugin-styled-components": "^3.0.0",
    "gatsby-source-drupal": "^3.0.8",
    "less": "^3.8.1",
    "less-loader": "^4.1.0",
    "prop-types": "^15.6.2",
    "react": "^16.5.1",
    "react-animated-css": "^1.0.4",
    "react-animated-slider": "^1.1.4",
    "react-coverflow": "^0.2.16",
    "react-dom": "^16.5.1",
    "react-helmet": "^5.2.0",
    "react-icons": "^3.2.2",
    "react-images": "^0.5.19",
    "react-loader-spinner": "^2.0.6",
    "react-measure": "^2.1.2",
    "react-photo-gallery": "^6.2.1",
    "react-spring": "^5.8.0",
    "react-typist": "^2.0.4",
    "react-waypoint": "^8.0.3",
    "styled-components": "^4.0.0-beta.8"
  }
}
```

**Comment by Carl-Maxwell** [comment]
Having the same problem with NPM. Just tried to install this package for the first time and on running npm install I got this:

```shell
$ npm install --save gatsby-plugin-favicon
npm ERR! Error while executing:
npm ERR! C:\Program Files\Git\mingw64\bin\git.EXE ls-remote -h -t git://github.com/Creatiwity/favicons-webpack-plugin.git
npm ERR!
npm ERR! fatal: unable to look up github.com (port 9418) (Either the application has not called WSAStartup, or WSAStartup failed. )
npm ERR!
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\carlm\AppData\Roaming\npm-cache\_logs\2018-12-17T13_02_24_789Z-debug.log
```

I'm on windows 10, here's the dependencies in my package.json:

```json
{
  "dependencies": {
    "gatsby": "^2.0.19",
    "gatsby-image": "^2.0.22",
    "gatsby-plugin-feed": "^2.0.8",
    "gatsby-plugin-google-analytics": "^2.0.5",
    "gatsby-plugin-manifest": "^2.0.5",
    "gatsby-plugin-offline": "^2.0.5",
    "gatsby-plugin-react-helmet": "^3.0.0",
    "gatsby-plugin-sharp": "^2.0.6",
    "gatsby-plugin-typography": "^2.2.0",
    "gatsby-remark-copy-linked-files": "^2.0.5",
    "gatsby-remark-images": "^2.0.4",
    "gatsby-remark-prismjs": "^3.0.0",
    "gatsby-remark-responsive-iframe": "^2.0.5",
    "gatsby-remark-smartypants": "^2.0.5",
    "gatsby-source-filesystem": "^2.0.2",
    "gatsby-transformer-remark": "^2.1.6",
    "gatsby-transformer-sharp": "^2.1.3",
    "prismjs": "^1.15.0",
    "react": "^16.5.1",
    "react-dom": "^16.5.1",
    "react-helmet": "^5.2.0",
    "react-typography": "^0.16.13",
    "typeface-merriweather": "0.0.43",
    "typeface-montserrat": "0.0.43",
    "typography": "^0.16.17",
    "typography-theme-wordpress-2016": "^0.15.10"
  },
  "devDependencies": {
    "eslint": "^4.19.1",
    "eslint-plugin-react": "^7.11.1",
    "prettier": "^1.14.2"
  }
}
```

I think if you grab the https://github.com/gatsbyjs/gatsby-starter-blog on a fresh install of an OS it'll presumably have the same issue.

Looking up code 128 it turns out it's an authentication issue, I think you need to change package to use the https url (so instead of `git://github.com/Creatiwity/favicons-webpack-plugin.git` it should be `https://github.com/Creatiwity/favicons-webpack-plugin.git`).

If I just try to access that url github denies permission:

```shell
$ git clone git@github.com:Creatiwity/favicons-webpack-plugin.git
Cloning into 'favicons-webpack-plugin'...
The authenticity of host 'github.com (192.30.255.113)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added 'github.com,192.30.255.113' (RSA) to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by julien1619** [comment]
Thanks all for the report, in fact it was fixed by #51 but not published. So I've just released the version 3.1.5 a few minutes ago. Feel free to reopen this bug if needed ;)
